tests/contracts/test_deposit.py

```diff
@@ -1,22 +1,32 @@
 import pytest
 
 
+def compute_merkle_root(w3, leaf_nodes):
+    assert len(leaf_nodes) >= 1
+    empty_node = b'\x00' * 32
+    child_nodes = leaf_nodes[:]
+    for i in range(32):
+        parent_nodes = []
+        if len(child_nodes) % 2 == 1:
+            child_nodes.append(empty_node)
+        for j in range(0, len(child_nodes), 2):
+            parent_nodes.append(w3.sha3(child_nodes[j] + child_nodes[j+1]))
+        child_nodes = parent_nodes
+    return child_nodes[0]
+
+
 @pytest.mark.parametrize(
     'success,amount_deposit',
     [
         (True, 32),
-        (False, 31),
-        (False, 33),
-        (False, 0)
+        (True, 1),
+        (False, 0),
+        (False, 33)
```

Comment:
Then this line would be:
```suggestion
        (False, MAX_DEPOSIT + 1)
```